### ISSUE TITLE
ascii ordinal not in range for INCO AIR(r) device User Agents.

### ISSUE THREAD

**Issue by chrisdl** [selwin/python-user-agents#44]
To Reproduce
```
from user_agents import parse
parse("Mozilla/5.0 (Linux; Android 4.4.2; INCO AIR® Build/KOT49H) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/30.0.0.0 Mobile Safari/537.36 Instagram 7.7.0 Android (19/4.4.2; 240dpi; 480x854; Inco Air/INCO; INCO AIR®; h3466; mt6582; es_US)")
```

**Comment by sdcooke** [comment]
I think updating the ua-parser library fixes this problem - I just updated from 0.4.1 to 0.6.1 and our test for this now passes.

**Comment by chrisdl** [comment]
yep, updating UA parser as per @sdcooke suggestion fixes the issue.
